### ISSUE TITLE
Copy Pods Resources fails

### ISSUE THREAD

**Issue by dnkoutso** [CocoaPods/CocoaPods#4548]
```
source 'https://github.com/CocoaPods/Specs'

target 'SampleApp' do 
	pod "Braintree", "4.0.0"
end
```

We make an internal release of CocoaPods for new features that we need. Here are the shas

```
  # CocoaPods master -> sha: 0f3eb52b3ec9a27635f872b53c5b5f99d5148447
  # Xcodeproj -> sha: fe9b173a46052a0743b4a7f52274f3bfd2ad327f
  # CocoaPods-Core -> sha: 6c2544496ed201104e712dd95793ec7bc6a171e8
```

Xcodeproj error:

```
PhaseScriptExecution Copy\ Pods\ Resources /Users/dimitris/Library/Developer/Xcode/DerivedData/SampleApp-fjulftzgkgfoslffkcevdzdsfhaf/Build/Intermediates/SampleApp.build/Debug-iphonesimulator/SampleApp.build/Script-2202DE3D46FAE9CD0B33B654.sh
    cd /Users/dimitris/Development/ios/SampleApp
    /bin/sh -c /Users/dimitris/Library/Developer/Xcode/DerivedData/SampleApp-fjulftzgkgfoslffkcevdzdsfhaf/Build/Intermediates/SampleApp.build/Debug-iphonesimulator/SampleApp.build/Script-2202DE3D46FAE9CD0B33B654.sh

error: Resource "/Users/dimitris/Development/ios/SampleApp/Pods//Users/dimitris/Library/Developer/Xcode/DerivedData/SampleApp-fjulftzgkgfoslffkcevdzdsfhaf/Build/Products/Debug-iphonesimulator/Braintree-UI-Localization.bundle" not found. Run 'pod install' to update the copy resources script.
```

**Comment by orta** [comment]
Awesome, but where does it fail? Can you provide a project file to download and look at please

**Comment by dnkoutso** [comment]
It fails when it runs the script. If you create a sample app with this pod and these shas you should get the failure.

The reason why I dont upload is because these shas are published internally for us so my `source` field would not work for you.

**Comment by segiddins** [comment]
@manuyavuz mind taking a quick look into this?

**Comment by manuyavuz** [comment]
@dnkoutso I'll take a look via trying with these shas when I'm available. Thanks for the report!

**Comment by manuyavuz** [comment]
I've found the problem, will fix today. Thanks for reporting @dnkoutso!

**Comment by dnkoutso** [comment]
No problem! Glad we are alpha testing ;)

**Comment by manuyavuz** [comment]
@dnkoutso, could you try the case again with updated repo? Do not forget to do a `pod install` first.

**Comment by dnkoutso** [comment]
Thank you will try today and report back!

**Comment by dnkoutso** [comment]
I can confirm the fix works! Our project builds again!

**Comment by manuyavuz** [comment]
Great! Thanks for alpha testing :+1:

**Comment by manuyavuz** [comment]
Closed via #4551
